src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -565,6 +566,166 @@ JSON example:
+| BolusCutOffDelayTime  | OPTIONAL, REQUIRED if `BolusCutOffFlag` is `true` | [number][] or [array][] of [numbers][] | Duration between the end of the labeling and the start of the bolus cut-off saturation pulse(s), in seconds. This can be a number or array of numbers, of which the values must be non-negative and monotonically increasing, depending on the number of bolus cut-off saturation pulses. For Q2TIPS, only the values for the first and last bolus cut-off saturation pulses are provided. Based on DICOM Tag 0018,925F `ASL Bolus Cut-off Delay Time`. |
+| BolusCutOffTechnique  | OPTIONAL, REQUIRED if `BolusCutOffFlag` is `true` | [string][]                             | Name of the technique used, for example Q2TIPS, QUIPSS, QUIPSSII. Corresponds to DICOM Tag 0018,925E `ASL Bolus Cut-off Technique`.                                                                                                                                                                                                                                                                                                                     |
+
+### `m0scan` metadata fields
+
+Some common metadata fields are required for the `*_m0scan.json`: `RepetitionTime` or `VolumeTiming`.
+
+| **Key name**         | **Requirement level** | **Data type**                          | **Description**                                                                                                                                                                                                                                                                                                                                                                   |
+|----------------------|-----------------------|----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| IntendedFor          | REQUIRED              | [string][] or [array][] of [strings][] | One or more filenames with paths relative to the subject subfolder, with forward slashes, referring to ASL time series for which the m0scan is intended.AcquisitionVoxelSize                                                                                                                                                                                                      |
```

Comment:
@effigies Same for IntendedFor. It has been used already in the specification. Now it has a paragraph and definition on the bottom of the page (Fieldmap Case 4). I can just refer to that one, I assume? Or should that paragraph (from fieldmap Case 4) go to common?